index.html

```diff
@@ -235,14 +235,15 @@ <h3><span><img src="images/Rectangle 57.png" alt=""></span>&nbsp; Skills</h3>
                 <p>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
             </div>
 
-            <form class="contact-form" action="">
-                <input class="full-name" type="text" placeholder="Full name">
+            <form class="contact-form" action="https://formspree.io/f/mayzrnlw" method="POST">
+                <input class="full-name" type="text" placeholder="Full name" maxlength="30" required>
                 <span>
-                <input class="first-name" type="text" placeholder="First Name"> <input class="second-name" type="text" placeholder="Last Name">
+                <input class="first-name" type="text" placeholder="First Name" maxlength="30" required>
+                <input class="second-name" type="text" placeholder="Last Name" maxlength="30" required>
```

Comment:
- [ ] How many input tags are we supposed to have here? I think it is 3 ma, the `name, email, and the message(text area)`, in your case I think you have 2 more additional input tags and currently causing the form not to submit, see? 
<img width="388" alt="1" src="https://github.com/iankiema/My-Portifolio-Personal/assets/77843050/9d717280-0353-4bf8-8634-6c96b4de2fa9">